presto-main/src/test/java/io/prestosql/sql/planner/TestLogicalPlanner.java

```diff
@@ -1526,6 +1527,19 @@ public void testSizeBasedSemiJoin()
                                                 values("T_A"))))));
     }
 
+    @Test
+    public void testExplainAnalyze()
+    {
+        assertPlan("EXPLAIN ANALYZE SELECT regionkey FROM nation ORDER BY nationkey",
+                output(
+                        node(ExplainAnalyzeNode.class,
+                                strictProject(ImmutableMap.of("regionkey", expression("regionkey")),
```

Comment:
This works, but is a bit indirect way of checking this.

Maybe, instead of adding ORDER BY and strictProject, 
you can have "strictTableScan" matcher?

Reply:
Thanks for the feedback. I have added the previous query with a strictTableScan matcher.